github/orgs.go

```diff
@@ -154,6 +154,25 @@ func (s *OrganizationsService) Get(ctx context.Context, org string) (*Organizati
 	return organization, resp, nil
 }
 
+// GetByID fetches an organization.
+//
+// Note: GetByID uses the undocumented GitHub API endpoint /organizations/:id.
```

Comment:
I'd suggest we don't do that for 2 reasons:

1. I have seen increasingly convincing evidence that this is a part of the GitHub REST API v3, albeit not explicitly/consistently documented, and it would not break because GitHub employees actively encourage its use and a lot of code relies on it.
2. The current version is consistent with documentation of `UsersService.GetByID` and `RepositoriesService.GetByID`.